app/routes/events/all-events/components/events-tags-refinement.component.tsx

```diff
@@ -63,3 +74,27 @@ export const EventsTagsRefinementList = () => {
     </div>
   );
 };
+
+export const EventsClearFiltersText = () => {
```

Comment:
I noticed there's no Clear Filters button for mobile in the Figma design. I went ahead and left a comment for Austin in Figma about it. Just noting it here, I think it would be good to have, really useful on desktop.